test/button/button-subtle.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+		<title>subtle button tests</title>
+
+		<script src="../../../@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script src="../../../mocha/mocha.js"></script>
+		<script src="../../../chai/chai.js"></script>
+		<script src="../../../@polymer/test-fixture/test-fixture.js"></script>
+		<script src="../../../wct-mocha/wct-mocha.js"></script>
+
+		<script type="module" src="../../components/button/button-subtle.js"></script>
+	</head>
+	<body>
+		<test-fixture id="basic">
+			<template>
+				<d2l-button-subtle text="Subtle Button"></d2l-button-subtle>
+			</template>
+		</test-fixture>
+		<test-fixture id="with-icon">
+			<template>
+				<d2l-button-subtle text="Subtle Button with Icon" icon="d2l-tier1:gear"></d2l-button-subtle>
+			</template>
+		</test-fixture>
+		<test-fixture id="disabled">
+			<template>
+				<d2l-button-subtle disabled text="Disabled Subtle Button"></d2l-button-subtle>
+			</template>
+		</test-fixture>
+
+		<script type="module">
+			import { runAxe } from '../../test-helpers/a11y-helper.js';
+
+			suite('subtle button', function() {
```

Comment:
Should we be consistent with `describe` vs `suite`, and `it` vs `test`?

Reply:
Yeah I think so, does anyone have a preference?  This is what I seem to be seeing more often in examples and READMEs

Reply:
In the visual-diff I was able to use `async`/`await` pretty nicely ([example](https://github.com/BrightspaceUI/core/blob/dbatiste/visual-diff/test/button/button-subtle-perceptual.js#L28)) in tests and setup functions.  Minor thing.  Not sure if the TDD equivalents support that.  My preference is for the BDD style by I am flexible.

Reply:
Updated to use `async`/`await` instead of `done`, looks like it works with the TDD style.  I really have no preference between the two styles, can change to whatever others want